sensor/common/networkflow/manager/manager_impl.go

```diff
@@ -438,6 +442,8 @@ func (m *networkFlowManager) enrichConnection(conn *connection, status *connStat
 		// Otherwise, check if the remote entity is actually a cluster entity.
 		lookupResults = m.clusterEntities.LookupByEndpoint(conn.remote)
 	}
+	log.Debugf("Connection is fresh: %t", isFresh)
+	log.Debugf("Connection lookupResults: %+v", lookupResults)
```

Comment:
Connection to external entity is added to the graph, when lookup results are empty:

```
manager_impl.go:420: Debug: Enriching connection: 4cc256c53d07:  -> 10.212.1.175:80 (tcp)
manager_impl.go:423: Debug: Found container for connection? true
manager_impl.go:434: Debug: Container for connection: client-dns, 4cc256c53d07
manager_impl.go:445: Debug: Connection is fresh: false
manager_impl.go:446: Debug: Connection lookupResults: []
```